crates/bevy_gizmos/src/lib.rs

```diff
@@ -17,6 +17,7 @@
 //! See the documentation on [`Gizmos`] for more examples.
 
 mod arrows;
+mod circles;
```

Comment:
This should probably be pub? Along with `arrows`. Currently I don't see any way to access these types externally.

Reply:
TIL it's possible to have a struct returned by a function which isn't exported anywhere

Reply:
Yeah, Rust's module privacy rules are bizarre, where you can create "sealed" traits and structs.

This got used to create unimplementable traits, so now they can't fix it.